packages/vm/src/evm/interpreter.ts

```diff
@@ -98,14 +100,13 @@ export default class Interpreter {
     // Iterate through the given ops until something breaks or we hit STOP
     while (this._runState.programCounter < this._runState.code.length) {
       const opCode = this._runState.code[this._runState.programCounter]
-
       if (
-        (opCode === 0x56 || opCode === 0x57 || opCode === 0x5e) &&
-        this._runState.validJumps.length === 0
+        !this._runState.jumpAnalysisDone &&
```

Comment:
Super nit but maybe we want to name this flag `shouldDoJumpAnalysis` which we initialize to `true`, this saves a NOT operation for each opcode.